pynest/examples/pynest-microcircuit/network_params.py

```diff
@@ -0,0 +1,192 @@
+
+    Arguements:
+    PSP_rel: standard deviation of the evoked synaptiv potential
+    number_of_pop: number of populations in the microcircuit
+
+    Returns:
+    Matrix of the postsynaptic potentials.
+    """
+    dim = number_of_pop
+    std_mat = np.zeros((dim, dim))
+    std_value_normal = PSP_rel
+    std_value_exception = 0.05
+    std_mat[:, :] = std_value_normal
+    std_mat[0, 2] = std_value_exception
+    return std_mat
+
+net_dict = {
+    # Neuron model
+    'neuron_model': 'iaf_psc_exp',
+    # The default recording device is the spike_detector. If you also
+    # want to record the membrane potentials of the neurons, add
+    # 'voltmeter' to the list.
+    'rec_dev': ['spike_detector', 'voltmeter'],
+    # Names of the simulated populations.
+    'populations': ['L23E', 'L23I', 'L4E', 'L4I', 'L5E', 'L5I', 'L6E', 'L6I'],
+    # Number of neurons in the different populations. The order of the
+    # elements corresponds to the names of the variable 'populations'.
+    'N_full': np.array([20683, 5834, 21915, 5479, 4850, 1065, 14395, 2948]),
+    # Mean rates of the different populations in the non-scaled version
+    # of the microcircuit. Necessary for the scaling of the network.
+    # The order corresponds to the order in 'populations'.
+    'full_mean_rates':
+        np.array([0.971, 2.868, 4.746, 5.396, 8.142, 9.078, 0.991, 7.523]),
+    # Connection probabilities. The first index corresponds to the targets
+    # and the second to the sources.
+    'conn_probs':
+        np.array(
+            [[0.1009, 0.1689, 0.0437, 0.0818, 0.0323, 0., 0.0076, 0.],
+             [0.1346, 0.1371, 0.0316, 0.0515, 0.0755, 0., 0.0042, 0.],
+             [0.0077, 0.0059, 0.0497, 0.135, 0.0067, 0.0003, 0.0453, 0.],
+             [0.0691, 0.0029, 0.0794, 0.1597, 0.0033, 0., 0.1057, 0.],
+             [0.1004, 0.0622, 0.0505, 0.0057, 0.0831, 0.3726, 0.0204, 0.],
+             [0.0548, 0.0269, 0.0257, 0.0022, 0.06, 0.3158, 0.0086, 0.],
+             [0.0156, 0.0066, 0.0211, 0.0166, 0.0572, 0.0197, 0.0396, 0.2252],
+             [0.0364, 0.001, 0.0034, 0.0005, 0.0277, 0.008, 0.0658, 0.1443]]),
+    # Number of external connections to the different populations.
+    # The order corresponds to the order in 'populations'.
+    'K_ext': np.array([1600, 1500, 2100, 1900, 2000, 1900, 2900, 2100]),
+    # Factor to scale the indegrees.
+    'K_scaling': 0.1,
+    # Factor to scale the number of neurons.
+    'N_scaling': 0.1,
+    # Amplitude of excitatory postsynaptic potential.
+    'PSP_e': 0.15,
+    # Standard deviation of the postsynaptic potential.
+    'PSP_sd': 0.1,
+    # PSP mean matrix
+    'PSP_mean_matrix': get_mean_PSP_matrix(0.15, -4, 8),
+    # PSP std matrix
+    'PSP_std_matrix': get_std_PSP_matrix(0.1, 8),
+    # mean delay matrix.
+    'mean_delay_matrix': get_mean_delays(1.5, 0.75, 8),
+    # std delay matrix.
+    'std_delay_matrix': get_std_delays(0.75, 0.375, 8),
+    # Rate of the poissonian spike generator.
```

Comment:
Define all values explicitly in the net_dict (e.g. `g` is missing).
I suggest to update the dictionary afterwards with the calculated dependent parameters.
Avoid to define the same variable twice like PSP_e and the first argument of PSP_mean_matrix().